src/inspector/node_string.cc

```diff
@@ -118,9 +119,7 @@ const uint8_t* CharactersUTF8(const String& s) {
 }
 
 size_t CharacterCount(const String& s) {
-  icu::UnicodeString utf16 =
-      icu::UnicodeString::fromUTF8(icu::StringPiece(s.data(), s.length()));
-  return utf16.countChar32();
```

Comment:
This is a different thing, to be clear; it's `utf32_length_from_utf8` in simdutf terms. `utf16_length_from_utf8` would correspond to `.length()` in ICU, not `.countChar32()`.

Did you check whether this is the right behavior here?

Reply:
Yes, you are right. utf32 is the correct function. I'm surprised that the tests are passing even with the current change.